masterlist.yaml

```
@@ -7110,6 +7110,22 @@ plugins:
         util: '[TES4Edit v4.0.3h (Hotfix 1)](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 6
 
+  - name: 'Daedric Lord( Armor|-Classic Bound Armor)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/16442/'
+        name: 'Daedric Lord Armor'
+  - name: 'Daedric Lord Armor.esp'
+    dirty:
+      - <<: *quickClean
+        crc: 0x0CB6A1E6
+        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 3
+        udr: 1
+  - name: 'Daedric Lord-Classic Bound Armor.esp'
```

This plugin only edits graphics & already has *Graphics* in its description, so it can be tagged with *Deactivate* & *NoMerge*.

```suggestion
  - name: 'Daedric Lord-Classic Bound Armor.esp'
    tag:
      - Deactivate
      - NoMerge
```